connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -316,6 +319,36 @@ message StatCrosstab {
   string col2 = 3;
 }
 
+// Replaces null values.
+// It will invoke 'Dataset.na.fill' (same as 'DataFrameNaFunctions.fill') to compute the results.
+// Following 3 parameter combinations are supported:
+//  1, 'values' only contains 1 item, 'cols' is empty:
+//    replaces null values in all type-matched columns.
+//  2, 'values' only contains 1 item, 'cols' is not empty:
+//    replaces null values in specified columns.
+//  3, 'values' contains more than 1 items, then 'cols' is required to have the same length:
+//    replaces each specified column with corresponding value.
+message NAFill {
+  // (Required) The input relation.
+  Relation input = 1;
+
+  // (Optional) Optional list of column names to consider.
+  repeated string cols = 2;
+
+  // (Required) Values to replace null values with. Should contains at least 1 item.
+  repeated Type values = 3;
+
+  // Available data types.
+  message Type {
```

Comment:
Could we probably use `Literal` to ship these values? We can call `eval` to take this out in JVM side.

Reply:
Or is this consistent with other API we added so far? Then I am good.

Reply:
I think this is the first method that only support several datatype.

> Could we probably use Literal to ship these values?

the question is do we need to limit the datatypes in proto message?

Reply:
Okay, I am fine with the current way for now.

Reply:
@HyukjinKwon thank you. There are also some other methods only supporting a subset of datetypes. If we prefer use `Literal` then I can make such change in the future.

Reply:
It seems more flexible to restrict the types in the server side. Once we relax the restriction in the future, people don't need to update the clients.

Reply:
@cloud-fan @HyukjinKwon make sense, let me update it

Reply:
updated